README.rst

```diff
@@ -1013,6 +1013,17 @@ Choose from the list of available rules:
 
   The correct case must be used for standard PHP types in phpdoc.
 
+* **phpdoc_types_order**
+
+  Sorts PHPDoc types.
+
+  Configuration options:
+
+  - ``null_position`` (``'always_first'``, ``'always_last'``, ``'default'``): forces the
+    position of ``null``; defaults to ``'always_first'``
+  - ``sort_algorithm`` (``'alpha'``, ``'none'``): the sorting algorithm to apply;
+    defaults to ``'none'``
```

Comment:
yep, I would keep it as that. if one want to have not-standardized order, he could configure the fixer or not use it. idea of fixing is to keep code as similar across whole project of given user as possible. for that, configuring to not update the code is another direction.